spec/factories/articles.rb

```diff
@@ -70,4 +71,10 @@
   trait :with_discussion_lock do
     after(:create) { |article| create(:discussion_lock, locking_user_id: article.user_id, article_id: article.id) }
   end
+
+  trait :past do
```

Comment:
I am a bit surprised that this is an after create hook.  Not that that is bad, you've spent time in this.  

Could you add a comment for when you should use the `:past` trait?  My read of the tests is that it's for when you want an article published sometime in the past.

Reply:
That's part of the "hack" that I mentioned [here](https://github.com/forem/forem/pull/17939#discussion_r901483566). Ideally, we'll remove this trait after the refactoring. I will add the comment in the factory too.